Formula/mockolo.rb

```diff
@@ -0,0 +1,60 @@
+class Mockolo < Formula
+  desc "Efficient Mock Generator for Swift"
+  homepage "https://github.com/uber/mockolo"
+  url "https://github.com/uber/mockolo/archive/1.1.5.tar.gz"
+  sha256 "53c828299311eb315b096e4d687811da49c0a45ab9647535c2879c144b016bbc"
+
+  pour_bottle? do
```

Comment:
This is not acceptable. Please don't specify a bottle block, the bot will add it.